tests/unit/test_clip.py

```diff
@@ -0,0 +1,8 @@
+from jina.executors import BaseExecutor
+
+def test_clip():
+    ex = BaseExecutor.load_config('../../config.yml')
```

Comment:
use `absolute path` or the relative path with this file as reference.

Adding `unittests` I would expect to actually have `encode` with different types of inputs. 

I would expect `unittests` to be more ambitious in catching edge cases